core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -35,9 +52,50 @@ import { IInteractionCustomizationArgs } from
 })
 export class ExplorationHtmlFormatterService {
   constructor(
-    private camelCaseToHyphens: CamelCaseToHyphensPipe,
-    private extensionTagAssembler: ExtensionTagAssemblerService,
-    private htmlEscaper: HtmlEscaperService) {}
+      private camelCaseToHyphens: CamelCaseToHyphensPipe,
+      private extensionTagAssembler: ExtensionTagAssemblerService,
+      private htmlEscaper: HtmlEscaperService
+  ) {}
+  /**
+   * Convert SubtitledHtml to html string and SubtitledUnicode to unicode
+   * string.
+   * @param caValues customization argument values
+   */
+  convertCustomizationArgsToCustomizationArgsHtml(
+      caValues: InteractionCustomizationArgs
+  ) : InteractionCustomizationArgsHtml {
+    if (!caValues) {
```

Comment:
Why would this case arise if the input argument is valid?

If it is not expected, perhaps throw an error instead?